.github/workflows/docs.yaml

```
@@ -0,0 +1,75 @@
+
+          with open("docs/api-reference/openapi.json", "w") as file:
+              json.dump(openapi_data, file, indent=4)
+          EOF
+
+      - name: Deploy to pages branch
+        run: |
+          git config --global user.name "Qreater AI"
+          git config --global user.email "qreater@qreater.com"
+          git add docs/api-reference/openapi.json
+          git commit -m "docs: introduce updates to openapi.json"
+          git push origin pages --force
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)